pkg/verifier/aws/aws_verifier.go

```diff
@@ -521,16 +522,33 @@ func (a *AwsVerifier) CreateSecurityGroup(ctx context.Context, tags map[string]s
 // ipPermissionFromURL generates an EC2 IpPermission (for use in sec. group rules) from a given IP address
 // -based http(s) URL (e.g. "https://10.0.8.1:8080") with the given human-readable description (ipPermDescription)
 func ipPermissionFromURL(ipUrlStr string, ipPermDescription string) (*ec2Types.IpPermission, error) {
-	// First validate URL by parsing it
+
+	//validate URL by parsing it
 	ipUrl, err := url.Parse(ipUrlStr)
```

Comment:
As this function is no longer dealing with just IP-address-based URLs, let's change the names of `ipUrlStr` and `ipUrl` to something like `allowedUrl[Str]` to avoid confusion

Reply:
I will for the first couple. however since in the end what we want to be dealing with even when we get a fully qualified domain is to work with IPs in order to add them to SGs, i will leave the name of variables towards the end as they are.